x-pack/packages/ml/anomaly_utils/types.ts

```diff
@@ -343,7 +343,7 @@ export interface MlAnomaliesTableRecord {
   /**
    * Returns true if the job has the model plot enabled
    */
-  modelPlotEnabled: boolean;
+  modelPlotEnabled?: boolean;
```

Comment:
I noticed this because for this PR I reused some mock data previously only used with `js` instead of `ts` and assigning the mock to this type failed. First I thought the mock was outdated, but it turns out looking at current live data we get from Kibana server this attribute isn't always present.